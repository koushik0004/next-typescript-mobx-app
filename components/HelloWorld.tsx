import React from 'react';

export const HelloWorld = ({}) => <div>Hello World</div>;

export default HelloWorld;
