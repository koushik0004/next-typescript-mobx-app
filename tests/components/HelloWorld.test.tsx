import React from 'react';
import renderer from 'react-test-renderer';
import HelloWorld from '../../components/HelloWorld';

describe('HelloWorld', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<HelloWorld />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
