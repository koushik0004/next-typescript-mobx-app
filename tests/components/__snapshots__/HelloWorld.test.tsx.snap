// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld should render correctly 1`] = `
<div>
  Hello World
</div>
`;
